Neos.Media/Classes/Domain/Repository/ThumbnailRepository.php

```diff
@@ -35,6 +39,12 @@ class ThumbnailRepository extends Repository
      */
     protected $entityManager;
 
+    /**
+     * @Flow\Inject
+     * @var LoggerInterface
+     */
+    protected $logger;
+
```

Comment:
This is rather a debugging tool – if this is considered working, it can be removed along with the logging calls below.

Reply:
I think those messages are actually useful, so let's keep those loging calls.